### PR TITLE
feat: add migration script to migrate org

### PR DESCRIPTION
**Pull request by sheensantoscapadngan**
# Description 📣
Script to dump database apart from the audit logs table, import into another Postgres instance and delete all data for organizations except one given organization.

Use `npm run migrate:org` in backend to test.

Note: 
- Ensure you run the import scripts on an empty postgres instance. 
- This can be done by just using `docker-compose -f <>.yml up db --build` and then using `npm run migrate:org` to import first.
- You use same `ENCRYPTION_KEY` and `ROOT_ENCRYPTION_KEY`

## Type ✨

- [ ] Bug fix
- [x] New feature
- [ ] Improvement
- [ ] Breaking change
- [ ] Documentation

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/getting-started/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/getting-started/code-of-conduct). 📝

### CODE REVIEW

**Comment by sheensantoscapadngan** [comment]
1. Move post-migration script to a transaction
2. Ensure that audit log table is created as well

**Comment by sheensantoscapadngan** [comment]
**Important Note**: When ran in production, make sure to use a DB replica/clone

File: backend/scripts/migrate-organization.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  execSync(
+    `PGDATABASE="${importDatabase}" PGPASSWORD="${importPassword}" PGHOST="${importHost}" PGPORT=${importPort} PGUSER=${importUser} psql -f ${path.join(
+      __dirname,
+      "../src/db/dump.sql"
+    )}`
+  );
+
+  execSync(
```
Comment (maidul98) [inline backend/scripts/migrate-organization.ts:48]: What about secrets? Permissions and all other resources? Are they removed because of cascade?

Comment (maidul98) [reply backend/scripts/migrate-organization.ts:48]: ** Remove by cascade

**Review by maidul98 — APPROVED** [review]
lgtm

**Pull request merged**